### ISSUE TITLE
Unable to debug with Chrome after updating to VS Code 1.95.1 - freezes if any breakpoint is set

### ISSUE THREAD

**Issue by estaylorco** [microsoft/vscode#233383]
Type: <b>Bug</b>

I'm using Aurelia v1.0 to develop a web application I've been working on for a long, long time.  I've had no problems with debugging until VS Code 1.95.1.  At the time I updated to 1.95.1, I had performed no NPM updates, or any other updates.

Also, I'm using WebPack.

Updating to VS Code 1.95.2 did not solve the problem, as I had hoped.

1. With a Chrome launch configuration, launch the debugger.
2. If any one breakpoint is already set, the url destination spin-waits.
3. If no breakpoints are set upon launching, the application loads and it works fine so long as no breakpoints are subsequently set.
4. Then, set a breakpoint.
5. Application will freeze if any action would cause the breakpoint to be accessed (not necessarily hit).

The last point is interesting.  I have a section of code that will execute when I click "New Pricing Line Item" in my application, which is where I set my breakpoint.  Even if I don't click that action, but simply try to load the section of code that contains the breakpoint, the application freezes.  It's as though the very loading of the code that contains the breakpoint causes the problem (not necessarily hitting the breakpoint).

I can reproduce this problem at will by following the steps above.

VS Code version: Code 1.95.2 (Universal) (e8653663e8840adaf45af01eab5c627a5af81807, 2024-11-07T11:07:22.054Z)
OS version: Darwin arm64 23.6.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M2 Max (12 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|1, 1, 1|
|Memory (System)|32.00GB (1.32GB free)|
|Process Argv|--crash-reporter-id 9cfcad7d-aa40-49e6-ad70-a4fe895ab775|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (16)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|13.5.0
copy-relative-path|ale|0.0.2
aurelia|Aur|2.1.4
Aurelia|beh|1.2.2
npm-intellisense|chr|1.4.5
path-intellisense|chr|2.9.0
vscode-eslint|dba|3.0.10
githistory|don|0.6.20
gitlens|eam|15.6.3
EditorConfig|Edi|0.16.4
prettier-vscode|esb|11.0.0
todo-tree|Gru|0.0.226
vscode-drawio|hed|1.6.6
vscode-css-navigation|puc|1.15.0
autoimport|ste|1.5.4
vscode-icons|vsc|12.9.0

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
9c06g630:31013171
dvdeprecation:31068756
dwnewjupytercf:31046870
newcmakeconfigv2:31071590
impr_priority:31102340
nativerepl1:31139838
refactort:31108082
pythonrstrctxt:31112756
cf971741:31144450
iacca1:31171482
notype1:31157159
5fd0e150:31155592
dwcopilot:31170013

```

</details>

**Comment by gjsjohnmurray** [comment]
/duplicate of https://github.com/microsoft/vscode/issues/232651 which is fixed in 1.95.2 that just released.

**Comment by estaylorco** [comment]
@gjsjohnmurray

This problem was not fixed with 1.95.2, as I indicated in my report.  I still cannot debug, and I can reproduce the problem at will, even after updated to 1.95.2.

**Comment by gjsjohnmurray** [comment]
> as I indicated in my report

#233432
